Updates to billing FAQs (#3911)
dbt-labs/docs.getdbt.com: website/docs/faqs/Accounts/cloud-upgrade-instructions.md

<details>
  <summary>How does billing work?</summary>
  <div>

>     <div>Team plans are billed monthly on the credit card used to sign up, based on [developer seat count and usage](/docs/cloud/billing). You’ll also be sent a monthly receipt to the billing email of your choice. You can change any billing information in your <b>Account Settings</b> -> <b>Billing page</b>. <br></br><br></br>

Reviewer: ```suggestion
    <div>Team plans are billed monthly on the credit card used to sign up, based on [developer seat count and usage](/docs/cloud/billing). You’ll also be sent a monthly receipt to the billing email of your choice. You can change any billing information in your <b>Account Settings</b> > <b>Billing page</b>. <br></br><br></br>
```

<details>
  <summary>Can I pay by invoice?</summary>
    <div>

>       <div>At present, dbt Cloud Team plan payments must be made via credit card, and by default they will be billed monthly based on the number of [developer seats and usage](/docs/cloud/billing). <br></br><br></br>

Reviewer: ```suggestion
      <div>Currently, dbt Cloud Team plan payments must be made with a credit card, and by default they will be billed monthly based on the number of [developer seats and usage](/docs/cloud/billing). <br></br><br></br>
```
